### ISSUE TITLE
FSR 2.2 

### ISSUE THREAD

**Issue by TheDiagnosed9mm** [turanszkij/WickedEngine#794]
I have had a go at implementing FSR 2.2 but I have failed. Are there any plans for FSR 2.2 or 3.

**Comment by turanszkij** [comment]
No plans yet from me.

**Comment by TheDarkThief** [comment]
Hi I was wondering if I were to submit a PR with this feature would it be accepted?
Seems like a fun challenge.

**Comment by turanszkij** [comment]
Yes, but it must be added without AMD's framework, like currently, without dependencies.
